### PR TITLE
technical/build_status: update outdated instructions

### PR DESCRIPTION
**Pull request by jakubgs**
Related to: https://github.com/status-im/status-react/pull/12580

Also did some cleanup to remove some boilerplate.

### CODE REVIEW

File: source/technical/build_status/index.md [hunk 1]
```diff
@@ -6,94 +6,89 @@ description: Build Status and participate in a Better Web
...
+- [GNU Make](https://www.gnu.org/software/make/)
 
-- Linux: `git`, `curl` and `make`
-  - Run `sudo apt install -y git curl make` on the terminal
-- macOS: Make sure you have `curl` installed
+
+On Debian/Ubuntu simply use:
+```sh
+sudo apt install -y git curl make
+```
```
Comment (Parveshdhull) [inline source/technical/build_status/index.md:20]: On Arch Linux simply use:
```sh
sudo pacman -S git curl make
```

File: source/technical/build_status/index.md [hunk 2]
```diff
@@ -6,94 +6,89 @@ description: Build Status and participate in a Better Web
...
+#### A. Clojure compiler
 
 In the first terminal window, just run:
 
-```bash
+```sh
 make run-clojure
 ```
 
 By doing this you will start the compilation of ClojureScript sources and run re-frisk (a tool for debugging). You should wait until it shows you `Build completed` before running the React Native packager.
```
Comment (Parveshdhull) [inline source/technical/build_status/index.md:73]: I am not sure but `run-clojure` doesn't run `run re-frisk` on own. We have to manually start that

**Review by Parveshdhull — APPROVED** [review]
Other than these small 3 changes, looks good to me 👍, Thanks

File: source/technical/build_status/index.md (hunk 2 above)
Comment (jakubgs) [reply source/technical/build_status/index.md:73]: Right, there is `make run-re-frisk` now.

File: source/technical/build_status/index.md (hunk 1 above)
Comment (jakubgs) [reply source/technical/build_status/index.md:20]: I don't intend to clutter up these instructions with instructions to every possible obscure distro.

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**